src/config/firefox-desktop.js

```diff
@@ -48,6 +48,9 @@ export default {
     if (pr.record_in_processes[0] === 'all') {
       pr.record_in_processes = ['main', 'content', 'gpu'];
     }
+    if (pr.record_in_processes[0] === 'all_childs') {
+      pr.record_in_processes = ['main', 'content'];
```

Comment:
This should be `['content', 'gpu']`. All childs is everything but main (aka parent).